pkg/pillar/utils/persist/persist.go

```diff
@@ -0,0 +1,22 @@
+package persist
```

Comment:
Please add a copyright and license line.

Reply:
Done